h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -1979,31 +1979,6 @@ private void scoreAndUpdateModel(){
       Frame train = DKV.<Frame>getGet(_parms._train); // need to keep this frame to get scoring metrics back
       _model.score(train).delete();
       scoreEnd(train, t1);
-/*
```

Comment:
why is this commented out? was it replaced by some other code somewhere else?

Reply:
The metrics computation is in scoreEnd now.